CardsCashCasino/Manager/PotManager.cs

```diff
@@ -0,0 +1,145 @@
+﻿using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics.Eventing.Reader;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+
+namespace CardsCashCasino.Manager
+{
+    /// <summary>
+    /// Manages the pot in a Texas Hold'em game, including adding and distributing chips.
+    /// </summary>
+    public class PotManager
```

Comment:
I want to rework this a bit. Not really a fan of a bunch of floating ints and pointers. Instead, lets make a Pot class that stores this info, and a list of those objects in HoldEmManager.

Reply:
Got it. Now implemented in HoldEmManager.